app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/BuyTicketActivity.java

```diff
@@ -81,7 +76,7 @@ public void run(){
                     // Jump to the payment activity
                     TicketType selectedType = getTicketTypeForName((String)ticketTypeSpinner.getSelectedItem());
                     Intent intent = new Intent(getApplicationContext(), StripePaymentActivity.class);
-                    intent.putExtra("ticketPrice", selectedType.getPrice());
+                    intent.putExtra("ticketPrice", selectedType.getPrice() / 100);
```

Comment:
`100.0` ? 
What about [formating](https://stackoverflow.com/questions/8819842/best-way-to-format-a-double-value-to-2-decimal-places)?